bin/rebuild.sh

```diff
@@ -0,0 +1,6 @@
+echo 'Stop and remove tus-php containers ...'
```

Comment:
missing shebang

```bash
#!/bin/sh
```

Also we need to change shebang to `/bin/sh` in other shell scripts